landing-page/src/ha-landing-page.ts

```diff
@@ -10,14 +10,14 @@ import { extractSearchParam } from "../../src/common/url/search-params";
 import { onBoardingStyles } from "../../src/onboarding/styles";
 import { makeDialogManager } from "../../src/dialogs/make-dialog-manager";
 import { LandingPageBaseElement } from "./landing-page-base-element";
-import { waitForSeconds } from "../../src/common/util/wait";
 import {
   getSupervisorNetworkInfo,
   pingSupervisor,
   type NetworkInfo,
 } from "./data/supervisor";
 
-export const CORE_CHECK_SECONDS = 5;
+export const ASSUME_CORE_START_SECONDS = 30;
+export const SCHEDULE_CORE_CHECK_SECONDS = 5;
```

Comment:
This is not used? We only check core status once now?

Reply:
We do check core status on every fail of `_fetchSupervisorInfo`, I changed it now that when we are in core check mode we run `_fetchSupervisorInfo` every second.